css-view-transitions-2/Overview.bs

```diff
@@ -1114,9 +1114,9 @@ When capturing the old or new state for an element, perform the following steps
 ### Compute the nearest containing 'view-transition-group' ### {#vt-group-nearest-contain}
 <div algorithm="nearest containing group name">
 
-To get the <dfn>nearest containing group name</dfn> for an {{Element}} |element|, perform the following steps:
+To get the <dfn>nearest containing group name</dfn> for an {{Element}} |element| and a [=tree=] |tree|, perform the following steps:
 
-	1. Let |nearestAncestorWithContain| be |element|'s nearest ancestor whose 'view-transition-name' [=computed value=] is not ''view-transition-name/none'' and whose 'view-transition-group' [=computed value=] is ''view-transition-group/contain''.
+	1. Let |nearestAncestorWithContain| be |element|'s nearest [=tree/ancestor=] in |tree|, whose 'view-transition-name' [=computed value=] is not ''view-transition-name/none'' and whose 'view-transition-group' [=computed value=] is not ''view-transition-group/normal''.
```

Comment:
I don't think we need to pass tree here? We can ignore the group if its computed value's tree scope doesn't match view-transition-name's tree scope.

Reply:
Done